mailpoet/assets/css/src/components-plugin/_welcome-wizard.scss

```diff
@@ -35,14 +41,15 @@
   align-items: center;
   display: flex;
   justify-content: center;
+  margin-top: 8px;
 
   @include respond-to(medium-screen) {
     flex-direction: column;
   }
 }
 
 .mailpoet-wizard-step-illustration {
-  margin-right: $grid-gap;
+  margin-right: $grid-gap-xl;
```

Comment:
The values of the grid variables defined in https://github.com/mailpoet/mailpoet/blob/9f790efbf0f032fc32e264dc33450286b362976e/mailpoet/assets/css/src/settings/_grid.scss don't match exactly the changes request by the designers. I don't know if we are still and should be following those grid values closely or not. Here I opted to use `$grid-gap-xl` which has a value of 48px, but the designer actually asked for 40px. In others places, I opted to use the value asked by the designer instead of using the grid variables. I should probably choose only one approach throughout this PR, but I'm not sure which one to choose.

Reply:
I think that this is not a question for a reviewer but for the ticket creator @pavel-mailpoet or maybe @NeosinneR

Reply:
I invited Shiva to GitHub to get her opinion on this. I'll ping her here once she accepts the invitation.

Reply:
@shivanarrthine what's your opinion on this?

Reply:
I'm good with changing from 40px to 48px to use the grid variable.